features/users/user_delete.feature

```diff
@@ -94,3 +94,26 @@ Scenario: Delete a user who has coauthored a work
   When I go to the works page
   Then I should see "otheruser"
     And I should not see "testuser"
+
+  Scenario: Can delete a user who has an empty series
+    Given I am logged in as "testuser"
+    When I set up a draft "Invalid" with fandom "%" as part of a series "Empty"
+      And I press "Post"
+    Then I should see "Sorry! We couldn't save this work because:"
```

Comment:
It took me a sec to realize these steps were here because trying and failing to create a work in a new series would result in an empty series being created. Could you put a comment here explaining that's what's going on?